com.sayonara-player.sayonara.json

```diff
@@ -0,0 +1,149 @@
+{
+    "app-id": "com.sayonaraplayer.sayonara",
```

Comment:
This doesn't even match the manifest name, hence the build failure.

Reply:
filenamed changed to suggested value in d1bb6c9